resources/api.json

```diff
@@ -0,0 +1,3 @@
+{
+  "client_id":"1018177563991-url06g1vfid7kveu6eb26rqkka73milf.apps.googleusercontent.com"
+}
```

Comment:
I don't think this information should be public knowledge. You might need to void this token and make a new one.